### PR TITLE
Get BatchNorm working end-to-end

### PR DESCRIPTION
**Pull request by prigoyal**
batchnorm works now 💃 👯‍♂️

### CODE REVIEW

File: torch/csrc/autograd/functions/toffee/batch_normalization.cpp [hunk 1]
```diff
@@ -25,14 +24,29 @@ void BatchNormForward::primspec(PrimSpecContext* ctx, jit::node_list inputs, jit
...
-  ADD_ATTR("momentum",f,momentum);
-
-  auto sm = "saved_mean"+std::to_string(ctx->batch_norm_count);
-  auto sv = "saved_var"+std::to_string(ctx->batch_norm_count);
+  ADD_ATTR("momentum",f,1-momentum);
+
+  auto typ = inputs.at(1)->type()->cast<torch::jit::TensorType>();
+  int64_t the_size = typ->sizes()[0];
+  std::stringstream ss;
+  ss << the_size << "_" << ctx->batch_norm_count;
```
Comment (zdevito) [inline torch/csrc/autograd/functions/toffee/batch_normalization.cpp:32]: This is a temporary workaround to recreate the missing save_mean/save_var tensors on the Caffe2 side.
